src/styles/quiz-card/quiz-card.html

```diff
@@ -0,0 +1,11 @@
+<section class="card">
+  <h2 class="card__heading">Frage</h2>
+  <p class="card__question">Wie viele Zähne hat ein Eisbär?</p>
+  <button class="card__button">Zeige mir die Antwort</button>
+  <ul class="card__tag-list">
+    <li card__tag-tag>32</li>
+    <li card__tag-tag>42</li>
+    <li card__tag-tag>52</li>
+    <li card__tag-tag>62</li>
```

Comment:
Did you mean: also, I'd probably name the class `card__tag`.
```suggestion
    <li class="card__tag">32</li>
    <li class="card__tag">42</li>
    <li class="card__tag">52</li>
    <li class="card__tag">62</li>
```

Reply:
It also looks a bit as if the tags are your answer. The tags should be an additional feature, kind of like hashtags. The answer would also be a `p`.

Reply:
Oh yes, you are right! Thank you